docs/current_docs/tensors.md

```diff
@@ -177,9 +177,23 @@ DoubleVertex CObservation = new GaussianVertex(C, 1);
 CObservation.observe(new double[]{6, 12});
 
 //Use algorithm to find MAP or posterior samples for A and/or B
-Optimizer optimizer = Optimizer.of(new BayesianNetwork(A.getConnectedGraph()));
+Optimizer optimizer = KeanuOptimizer.of(new BayesianNetwork(A.getConnectedGraph()));
 optimizer.maxAPosteriori();
 
 //Retrieve the most likely estimate using MAP estimation
 DoubleTensor mostLikelyEstimate = A.getValue(); //approximately [2, 3]
 ```
+
+## Python
+
+We do not expose Tensors in the Python API. Numpy provides the same concept through its `ndarray` class. 
```

Comment:
through its `ndarray` class and these are converted to Tensors for you.

Reply:
Do we need to mention this though? It's completely hidden from the user.

Reply:
Aye, as people will be familiar with Tensors so it's good to ensure they know the translation is happening behind the scenes. `ndarray` is just another face on top of them